python3/templates/debian

```diff
@@ -36,81 +52,90 @@ def run(cmd, *args):
     debug("+ " + cmd % args)
     (ret, out) = commands.getstatusoutput(cmd % args)
     if verbose:
-        try: 
+        try:
             for line in out.split("\n"):
                 log("| " + line)
         except TypeError as e:
             pass
     if ret != 0:
-        debug ("run - command %s failed with %d" , cmd, ret)
+        debug("run - command %s failed with %d", cmd, ret)
         raise CommandException(out)
     return out
 
+
 def log(fmt, *args):
     print(fmt % args)
 
+
 def debug(msg, *args):
     if verbose:
         print(msg % args)
 
+
 def create_partition(lvpath):
     # 1. write a partition table:
-    pipe = os.popen('/sbin/fdisk %s' % lvpath, 'w')
+    pipe = os.popen("/sbin/fdisk %s" % lvpath, "w")
 
-    pipe.write('n\n') # new partition
-    pipe.write('p\n') # primary
-    pipe.write("1\n") # 1st partition
-    pipe.write('\n')  # default start cylinder
-    pipe.write('\n')  # size: as big as image
-    pipe.write('w\n') # write partition table
+    pipe.write("n\n")  # new partition
+    pipe.write("p\n")  # primary
+    pipe.write("1\n")  # 1st partition
+    pipe.write("\n")  # default start cylinder
+    pipe.write("\n")  # size: as big as image
+    pipe.write("w\n")  # write partition table
 
     # XXX we must ignore certain errors here as fdisk will
     # sometimes return non-zero signalling error conditions
     # we don't care about.  Should fix to detect these cases
     # specifically.
     rc = pipe.close()
-    if rc == None: 
+    if rc == None:
         rc = 0
         log("fdisk exited with rc %d (some non-zero exits can be ignored safely)." % rc)
 
+
 def map_partitions(lvpath):
     run("/sbin/kpartx -a %s", lvpath)
     ps = []
     for line in run("/sbin/kpartx -l %s" % lvpath).split("\n"):
         ps.append("/dev/mapper/" + line.split()[0])
     return ps
 
+
 def unmap_partitions(lvpath):
     run("/sbin/kpartx -d %s", lvpath)
 
+
 def umount(mountpoint):
-    run("umount -l %s",mountpoint)
+    run("umount -l %s", mountpoint)
+
 
 if __name__ == "__main__":
-    #os.setpgrp()
+    # os.setpgrp()
     xvda = os.getenv("xvda")
     xvdb = os.getenv("xvdb")
     debug("Guest's xvda is on %s" % xvda)
     debug("Guest's xvdb is on %s" % xvdb)
     if xvda == None or xvdb == None:
-        raise "Need to pass in device names for xvda and xvdb through the environment"
-    
+        raise ValueError ("Need to pass in device names for xvda and xvdb through the environment")
```

Comment:
This is the only line that isn't changed to conform to `black` (no space before the bracket, and `black` would be this:
```suggestion
        raise ValueError(
            "Need to pass in device names for xvda and xvdb through the environment"
        )
```
But see my other comments below on asking the developers if these templates are still used.

Reply:
During CI test for CP-47869 , had to Revert "CP-47869: Removed debian from scripts/templates/"
This reverts commit https://github.com/xapi-project/xen-api/commit/515a8b2e3da21e584a123960d14601ea69538a92 as debian script is used during CI test .